extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -0,0 +1,104 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { ObjectManagementDialogBase, ObjectManagementDialogOptions } from './objectManagementDialogBase';
+import { IObjectManagementService, ObjectManagement } from 'mssql';
+import * as localizedConstants from '../localizedConstants';
+import { CreateDatabaseDocUrl } from '../constants';
+import { DefaultInputWidth } from './dialogBase';
+
+export class DatabaseDialog extends ObjectManagementDialogBase<ObjectManagement.Database, ObjectManagement.DatabaseViewInfo> {
+	private _nameInput: azdata.InputBoxComponent;
+
+	constructor(objectManagementService: IObjectManagementService, options: ObjectManagementDialogOptions) {
+		super(objectManagementService, options);
+	}
+
+	protected override get docUrl(): string {
+		return CreateDatabaseDocUrl;
+	}
+
+	protected override async validateInput(): Promise<string[]> {
+		const errors: string[] = [];
+		if (!this.objectInfo.name) {
+			errors.push(localizedConstants.NameCannotBeEmptyError);
+		} else if (this.viewInfo.databaseNames.some(name => name.toLowerCase() === this.objectInfo.name.toLowerCase())) {
```

Comment:
I am not sure whether you should do case insensitive check, I think it is depending on the server's collation.  I'd delegate this to server (don't do the check and let the engine block it when executing the query)

Reply:
Like skip checking for any database names at all?

Reply:
Removed the name validation here.